webonary-cloud-api/tools/flexXhtmlParser.ts

```diff
@@ -202,41 +195,45 @@ export class FlexXhtmlParser {
       [],
     );
 
-    const data: EntryData = {
+    const entry: DictionaryEntry = {
+      _id,
       dictionaryId,
       mainHeadWord,
       letterHead,
       pronunciations,
-      senses: { partOfSpeech, definitionOrGloss, semanticDomains },
+      senses: [{ definitionOrGloss, semanticDomains }],
```

Comment:
partOfSpeech should be added back in here.

Reply:
We decided to address this in a followup PR as there are more changes in this area.